src/doc/man/cargo-metadata.md

```diff
@@ -318,8 +318,10 @@ The JSON output has the following format:
         */
         "root": "file:///path/to/my-package#0.1.0",
     },
-    /* The absolute path to the build directory where Cargo places its output. */
+    /* The absolute path to the target directory where Cargo places its output. */
     "target_directory": "/path/to/my-package/target",
+    /* The absolute path to the build directory where Cargo places intermediate build artifacts. */
```

Comment:
Could we put an `(unstable)` at the end of this?

Reply:
Just like other CLI help texts. If the doc is going to show up in the stable channel, we do note them :)